src/test/java/uk/gov/dhsc/htbhf/steps/NavigationSteps.java

```diff
@@ -36,7 +36,7 @@ public void navigateToPage(String pageName) {
         page.openDirect();
     }
 
-    @Then("^I am shown the (.*) page")
+    @Then("^I am shown the (.*) page$")
```

Comment:
I initially added this as the original step matched this with extra words after `page` so the dollar makes it an exact match - it doesn't hurt to keep this in.